lib/MusicBrainz/Server/Controller/ReleaseGroup.pm

```diff
@@ -30,6 +30,10 @@ with 'MusicBrainz::Server::Controller::Role::EditRelationships';
 with 'MusicBrainz::Server::Controller::Role::JSONLD' => {
     endpoints => {show => {copy_stash => [{from => 'releases_jsonld', to => 'releases'}]}, aliases => {copy_stash => ['aliases']}}
 };
+with 'MusicBrainz::Server::Controller::Role::Collection' => {
+    entity_name => 'rg',
```

Comment:
This is a dumb distinction I'd like to remove later. Pretty sure release groups are the only entity type that does this (uses a different name for the entity in the stash).